### PR TITLE
Feature - add Azure Event Hubs dependency tracking

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #59

### CODE REVIEW

File: src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs [hunk 1]
```diff
@@ -340,9 +340,50 @@ public static void LogTableStorageDependency(this ILogger logger, string account
...
+        ///     Logs an Azure Event Hub Dependency.
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="accountName">Account of the resource</param>
+        /// <param name="eventHubName">Name of the Event Hub resource</param>
+        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
+        /// <param name="startTime">Point in time when the interaction with the dependency was started</param>
+        /// <param name="duration">Duration of the operation</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogEventHubsDependency(this ILogger logger, string accountName, string eventHubName, bool isSuccessful, DateTimeOffset startTime, TimeSpan duration, Dictionary<string, object> context = null)
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs:376]: Event Hubs uses the concept of namespaces similar to Service Bus, so we should name it accordingly.
```suggestion
        public static void LogEventHubsDependency(this ILogger logger, string namespaceName, string eventHubName, bool isSuccessful, DateTimeOffset startTime, TimeSpan duration, Dictionary<string, object> context = null)
```

Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs:376]: And can I use the `namespaceName` as dependency data?

Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs]: Updated.

**Comment by stijnmoreels** [comment]
image

File: src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs (hunk 1 above)
Comment (tomkerkhove) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs]: Sure!

**Pull request merged**